genesis/camino_unparsed_config.go

```diff
@@ -251,18 +264,28 @@ func (udo UnparsedDepositOffer) Parse(startTime uint64) (genesis.DepositOffer, e
 		do.Flags |= deposit.OfferFlagLocked
 	}
 
+	if offerID, err := ids.FromString(udo.OfferID); udo.OfferID != "" && err != nil {
```

Comment:
in which case we won't have offerID in json?

Reply:
Templates (see prev answer)

Reply:
Leaving as is - but we have an idea to use the relative time in offerID calc...